src/content/docs/infrastructure/infrastructure-ui-pages/hosts-new-view.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Infrastructure Hosts page (new view)
+tags:
+  - Infrastructure
+  - Infrastructure monitoring UI
+  - Infrastructure UI
+  - Hosts page
+metaDescription: Use the Hosts page to view the most important metrics from across your server/host environment.
+---
+
+import infrastructureHostsNewSummary from 'images/infrastructure_screenshot-full_hosts-new-summary.png'
+
+import infrastructureSavedViewsBttn from 'images/infrastructure_icon_saved-views-bttn.png'
+
+import infrastructureExportCSV from 'images/infrastructure_icon_export-csv.png'
+
+import infrastructureHostNewSystem from 'images/infrastructure_screenshot-full_host-new-system.png'
+
+import infrastructureHostNewNetwork from 'images/infrastructure_screenshot-full_host-new-network.png'
+
+import infrastructureHostNewProcesses from 'images/infrastructure_screenshot-full_host-new-processes.png'
+
+import infrastructureHostNewStorage from 'images/infrastructure_screenshot-full_host-new-storage.png'
+
+import infrastructureHostNewContainers from 'images/infrastructure_screenshot-full_host-new-containers.png'
+
+
+<Callout title="PREVIEW FEATURE">
+    This feature is currently in preview. If you want to try it out, contact your New Relic representative.
+</Callout>
+
+Use the **Hosts** page to better understand important performance metrics, like resource usage, network and processes performance, and data about your containers. You can filter your hosts by attributes and compare their performance with inventory change events to help determine root causes. You can also set alert notifications to help prevent problems.
+
+## Access the new infrastructure UI [#access-new-ui]
+
+To view your hosts information, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Hosts (New)**.
+
+<img
+  title="Hosts (New) Summary page"
+  alt="Hosts (New) Summary page"
+  src={infrastructureHostsNewSummary}
+/>
+
+<figcaption>
```

Comment:
@x8a you might add "All Capabilities" into the link name just to make it explicit so users learn about this path.